components/ShopBuilder/ModalProducts.js

```diff
@@ -21,16 +23,42 @@ const Modal = styled.div`
 `;
 
 const ModalProducts = (props) => {
+  function handlFiles(e) {
+    props.setImageAction(
+      URL.createObjectURL(e.target.files[0]),
+      Number(props.editId)
+    );
+  }
+
+  function editModalType() {
+    switch (props.editType) {
+      case "image":
+        return (
+          <label>
+            Choose File:
+            <input
+              type="file"
+              accept="image/png, image/jpeg"
+              name={props.editId}
+              onChange={(e) => handlFiles(e)}
+            />
+          </label>
+        );
+      default:
+        return <div>broken</div>;
```

Comment:
Try to update this message to reflect to the under a more clear reason at why the process is failing and what to do next.